### PR TITLE
update ug for view, list and viewdepartment

### LINKED ISSUE #154: UG: Update FAQ
FAQ is quite lacking. Add in more common issues that users may face.

### PR DESCRIPTION
**Pull request by ngshijun**
Resolves #154

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -357,20 +361,50 @@ Apart from supporting the core HR functions, Coydir also supports *department-le
...
+- Research and Development
+- Sales
+- Technology
 
 #### View details of a department: `viewdepartment`
 
 View the summarized details of a department given the name of the department.
 
-Details include the number of employee in that particular department, employees who are currently available, employees who are currently on leave, and a table of employees in that department and their corresponding performance ratings.
+Details include the number of employee in that particular department, employees who are currently available, employees who are currently on leave, and a table of employees in that department with their corresponding performance ratings and availability.
```
Comment (prit3010) [inline docs/UserGuide.md:386]: Can add a picture of how viewdepartment looks like

File: docs/UserGuide.md [hunk 2]
```diff
@@ -357,20 +361,50 @@ Apart from supporting the core HR functions, Coydir also supports *department-le
...
+- Product Management
+- Production
+- Quality Assurance
+- Research and Development
+- Sales
+- Technology
 
 #### View details of a department: `viewdepartment`
 
 View the summarized details of a department given the name of the department.
```
Comment (prit3010) [inline docs/UserGuide.md:384]: Can explain a bit more on why we need `viewdepartment`. Maybe can say that as a department head, you will need to track the ratings of your employees for talent development and see how many people are in for manpower allocation

**Review by prit3010 — CHANGES_REQUESTED** [review]
Can be a bit more detailed in  the view department.

**Review by chao890 — CHANGES_REQUESTED** [review]
Can rename all instances of `viewdepartment` to `view-department`?

**Review by chao890 — APPROVED** [review]

**Review by prit3010 — APPROVED** [review]
LGTM

**Pull request merged**